test/onnx/test_fx_op_consistency.py

```diff
@@ -391,12 +391,12 @@
     ),
     xfail(
         "nn.functional.max_pool2d",
-        dtypes=onnx_test_common.BOOL_TYPES + onnx_test_common.INT_TYPES,
+        dtypes=onnx_test_common.BOOL_TYPES + (torch.uint8, torch.int16, torch.int32, torch.int64),
         reason=onnx_test_common.reason_onnx_does_not_support("Max_pool2d"),
     ),
     xfail(
         "nn.functional.max_pool3d",
-        dtypes=onnx_test_common.BOOL_TYPES + onnx_test_common.INT_TYPES,
+        dtypes=(torch.uint8, torch.int16, torch.int32, torch.int64),
```

Comment:
same here, but also for bools

Reply:
Same because it now passes the tests

Reply:
There aren't any bool types invocations for max_pool*d which is why it doesn't matter if it's there or not